website/docs/docs/build/projects.md

```diff
@@ -3,88 +3,69 @@ title: "About dbt projects"
+| Resource  | Description  |
+| :--- | :--- |
+| [models](/docs/build/models) | Each model lives in a single file and contains logic that either transforms raw data into a dataset that is ready for analytics or, more often, is an intermediate step in such a transformation. |
+| [snapshots](/docs/build/snapshots) | A way to capture the state of your mutable tables so you can refer to it later. |
+| [seeds](/docs/build/seeds) | CSV files with static data that you can load into your data platform with dbt. |
+| [tests](/docs/build/tests) | SQL queries that you can write to test the models and resources in your project. |
+| [macros](/docs/build/jinja-macros) | Blocks of code that you can reuse multiple times. |
+| [docs](/docs/collaborate/documentation) | Docs for your project that you can build. |
+| [sources](/docs/build/sources) | A way to name and describe the data loaded into your warehouse by your Extract and Load tools. |
+| [exposures](/docs/build/exposures) | A way to define and describe a downstream use of your project. |
+| [metrics](/docs/build/metrics) | A way for you to define metrics for your project. |
+| [analysis](/docs/build/analyses) | A way to organize analytical SQL queries in your project such as the general ledger from your QuickBooks. |
 
-## Creating a dbt project
+When building out the structure of your project, you should consider these impacts to your organization's workflow:
 
-:::info Creating your first dbt project
+* **How would people run dbt commands** &mdash; Selecting a path
+* **How would people navigate within the project** &mdash; Whether as developers in the IDE or stakeholders from the docs
+* **How would people configure the models** &mdash; Some bulk configurations are easier done at the directory level so people don’t have to remember to do everything in a config block with each new model
 
-If you're new to dbt, we recommend that you check out our [Getting Started guide](/docs/develop/getting-started) to build your first dbt project.
+## Project configuration
+Every dbt project includes a project configuration file called `dbt_project.yml`. It defines the directory of the dbt project and other project configurations.
 
-:::
+Edit `dbt_project.yml` to set up common project configurations such as:
 
-If you don't yet have a dbt project, follow these instructions to create one. The dbt [starter project](https://github.com/dbt-labs/dbt-starter-project) contains default configurations as well as helpful notes.
+<div align="center">
 
+| YAML key  | Value description  |
+| :--- | :--- |
+| [name](/reference/project-configs/name) | Your project’s name in [snake case](https://en.wikipedia.org/wiki/Snake_case) |
+| [version](/reference/project-configs/version) | The [dbt Core version](/docs/dbt-versions/core) your project is using |
```

Comment:
Although [`version` is a required parameter](https://deploy-preview-2190--docs-getdbt-com.netlify.app/reference/project-configs/version) within `dbt_project.yml`, "it is not currently meaningfully used by dbt"!

It can be _anything_ the user chooses as long as it is in `number.number.number` format. It literally won't affect anything no matter what they put.

So I'd suggest a clarification similar to the following:
```suggestion
| [version](/reference/project-configs/version) | Version of your project |
| [require-dbt-version](/reference/project-configs/require-dbt-version) | Restrict your project to only work with a range of [dbt Core versions](/docs/dbt-versions/core) |
```


Up to you if you actually want to include a row for `require-dbt-version` or not.